source/_components/tradfri.markdown

```diff
@@ -38,3 +38,7 @@ Configuration variables:
 <p class='note'>
 Do not use the `api_key` variable. The key is only needed once at initial setup.
 </p>
+
+<p class='note'>
+Please make sure you have autoconf installed (```apt-get install autoconf```) if you want to use this component. Also, installing some dependencies might take considerable time (>1h) on slow devices. You might have to use ```sudo``` when installing autoconf.
```

Comment:
Shouldn't inline code be formatted using single backtics?

Reply:
Missed that part of the standards. Have changed now.